src/ontology/mondo-edit.obo

```diff
@@ -245209,17 +245210,21 @@ synonym: "progressive epilepsy - intellectual disability, Finnish type" RELATED
 synonym: "progressive epilepsy with intellectual disability, northern epilepsy" EXACT [DOID:0110724]
 synonym: "progressive epilepsy with mental retardation, northern epilepsy" EXACT DEPRECATED [DOID:0110724]
 synonym: "progressive epilepsy-intellectual disability syndrome, Finnish type" EXACT [DOID:0110724, GARD:0004010]
+synonym: "progressive myoclonic epilepsy with neuroserpin inclusion bodies" EXACT [MONDO:0035160]
```

Comment:
Where did this synonym come from? I don't see it on `MONDO:0035160`?

Reply:
that was the name/label of the term that we merged in

Reply:
Oh right! I need to update the [docs](https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/) to include that obvious statement.